app/experimenter/nimbus-ui/src/components/FormBranches/index.tsx

```diff
@@ -238,6 +240,8 @@ export const FormBranches = ({
                       branch,
                       reviewErrors: reviewErrors?.["treatment_branches"]?.[idx],
                       onRemove: handleRemoveBranch(idx),
+                      defaultValues:
+                        defaultValues.treatmentBranches?.[idx] || {},
```

Comment:
I get:

```
Types of property 'defaultValues' are incompatible.
    Type 'AnnotatedBranch | null' is not assignable to type 'Record<string, any> | undefined'.
      Type 'null' is not assignable to type 'Record<string, any> | undefined
```

I'd have to accept `undefined`, `null`, or `Record<string, any>` rather than just `Record<string, any>`, so I think I'll leave this one as-is. 🤔